job-server/test/spark.jobserver/JobManagerSpec.scala

```diff
@@ -208,5 +210,26 @@ abstract class JobManagerSpec extends JobSpecBase(JobManagerSpec.getNewSystem) {
       }
     }
 
+    it("should be able to start a job with job jar dependencies and return results"){
+      manager ! JobManagerActor.Initialize(daoActor, None)
+      expectMsgClass(initMsgWait, classOf[JobManagerActor.Initialized])
+
+      uploadTestJar()
+      val extrasJarPath = Paths.get(".").toAbsolutePath().normalize().toString() +
+        "/../job-server-extras/target/scala-2.10/job-server-extras_2.10-0.6.2-SNAPSHOT.jar"
```

Comment:
@amarouni can you try using the `TestJarFinder` class?  It is designed to specifically work around this absolute path problem.  Pretty sure it (or a slight modification thereof) should help get around this problem.